pkg/server/event_handler.go

```diff
@@ -336,6 +323,14 @@ type notificationPayload struct {
 	Metadata map[string]string `json:"metadata,omitempty"`
 }
 
+func parsePmapEvent(event *v1alpha1.PmapEvent) ([]byte, error) {
+	eventByte, err := protojson.Marshal(event)
+	if err != nil {
+		return nil, fmt.Errorf("failed to marshal event to byte: %w", err)
+	}
+	return eventByte, nil
+}
```

Comment:
Don't see the point of having this helper. You're introducing more lines than w/o it